src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -168,6 +171,11 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 					reloading(sender, "aliases", logHandler);
 					Aliases.clear();
 					Aliases.loadAsync().thenRun(() -> reloaded(sender, logHandler, timingLogHandler, "aliases"));
+				} else if (args[1].equalsIgnoreCase("global-options")) {
```

Comment:
Is the idea that there might be more "globals" in the future (e.g. experiments)? Maybe it should just be like `/sk reload globals`?

Reply:
If we were to do this I would like to set the precedent that reloading globals forces a reload of all scripts.
This is a good location to lay the groundwork for custom types, which i've been experimenting with in oopsk, and those and similar sorts of global data (like experiments) are not particularly safe when scripts get out of sync in regards to parsed information. For options it's safe enough to only have a warning, but if we want to expand this I think we need to enforce a full reload when globals are changed.